pontoon/base/models.py

```diff
@@ -531,6 +510,23 @@ def create_user_profile(sender, instance, created, **kwargs):
 post_save.connect(create_user_profile, sender=User)
 
 
+def get_translation_dict(entity, translations_dict, i=None):
+    translation = {}
+
+    if entity in translations_dict:
+        translations = translations_dict[entity]
+        if i:
+            translations = [x for x in translations if x['plural_form'] == i]
+        translation = sorted(translations, key=lambda k: (-k['approved'], k['date']))[0]
+
+    return {
+        'fuzzy': translation.get('fuzzy', False),
```

Comment:
This `get` stuff is kind of unclear; your intent is to return a default value if the requested entity isn't in `translations_dict`, so you might as well just check for it directly instead of using `get` to handle that case:

```python
if entity not in translations_dict:
    # return a dict with all the default values being passed to get
else:
    #return the real dict
```